.github/workflows/publish.yml

```diff
@@ -0,0 +1,35 @@
+name: Publish Docker image
```

Comment:
Could you please just keep the `Dockerfile` in the PR? I will probably try to setup publishing to ghcr.io instead, to keep everything on GH, and I'll experiment with that in a separate PR, since I'll need to setup some permissions etc.